test/models/test_calculator.jl

```diff
@@ -26,18 +26,46 @@ elements = (:Si, :O)
 level = M.TotalDegree()
 max_level = 15
 order = 3 
+E0s = (-158.54496821, -2042.0330099956639)
+NZ = length(elements)
+
+function make_rin0cut(zi, zj) 
+   r0 = ACE1x.get_r0(zi, zj)
+   return (rin = 0.0, r0 = r0, rcut = 6.5)
+end
+
+rin0cuts = SMatrix{NZ, NZ}([make_rin0cut(zi, zj) for zi in elements, zj in elements])
```

Comment:
We should provide better convenience functions for this. But that can be a separate PR. Maybe file an issue and we can discuss what it should look like...